README.md

```diff
@@ -7,24 +7,49 @@ The PyFingerprint library allows to use ZhianTec ZFM-20, ZFM-60, ZFM-70 and ZFM-
 **Note:** The library is inspired by the C++ library from Adafruit Industries:
 <https://github.com/adafruit/Adafruit-Fingerprint-Sensor-Library>
 
-## Package building on Debian
+## Installation
+
+There are two ways of installing PyFingerprint: Installation of the stable or latest version. The stable version is distributed through the PM Code Works APT repository and is fully tested but does not contain the latest changes.
+
+### Installation of the stable version
+
+Add PM Code Works repository
+
+* Debian 9:
```

Comment:
We should think about supporting the 18/20er Ubuntu versions because most users do not have Debian on their dev machines